tests/runfiles/linux.run

```diff
@@ -873,3 +873,14 @@ tags = ['functional', 'zvol', 'zvol_swap']
 [tests/functional/libzfs]
 tests = ['many_fds', 'libzfs_input']
 tags = ['functional', 'libzfs']
+
+
+# Unit tests are separate from functional tests because they cover the
```

Comment:
For these test groups, if you add `pre =` and `post =` then you can omit the setup and cleanup files, which aren't doing anything at the moment.

Reply:
good idea, less is more